.github/workflows/test-microk8s.yaml

```diff
@@ -0,0 +1,83 @@
+name: Charmed Katib
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        set -eux
+        sudo snap install microk8s --classic
```

Comment:
Since you are not using any pre-existing GitHub actions (e.g. `uses: actions/checkout@v2`) to install microk8s, does GitHub actions have any advantages  compare to Travis ?
What do you think @gaocegege @johnugeorge @knkski ?

Reply:
https://github.com/helm/kind-action We can use this action, I think.

Reply:
Updated it to use the `balchua/microk8s-actions@v0.2.1` action. I have a slight preference for github actions due to the UI being better about scrolling through large amounts of output, but can switch to Travis CI if desired.